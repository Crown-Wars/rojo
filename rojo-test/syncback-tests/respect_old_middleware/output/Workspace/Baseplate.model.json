{
  "className": "Part",
  "children": [
    {
      "className": "Texture",
      "name": "Texture"
    }
  ]
}