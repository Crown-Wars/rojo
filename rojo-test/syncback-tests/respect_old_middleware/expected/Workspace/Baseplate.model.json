{
  "className": "Part",
  "children": [
    {
      "name": "Texture",
      "className": "Texture",
      "properties": {
        "Color3": [
          0.0,
          0.0,
          0.0
        ],
        "Face": "Top",
        "StudsPerTileU": 8.0,
        "StudsPerTileV": 8.0,
        "Texture": "rbxassetid://6372755229",
        "Transparency": 0.800000011920929
      }
    }
  ],
  "properties": {
    "Anchored": true,
    "BottomSurface": "Smooth",
    "CFrame": [
      0.0,
      -8.0,
      0.0,
      1.0,
      0.0,
      0.0,
      0.0,
      1.0,
      0.0,
      0.0,
      0.0,
      1.0
    ],
    "Color": {
      "Color3uint8": [
        91,
        91,
        91
      ]
    },
    "FormFactor": "Symmetric",
    "Locked": true,
    "Size": [
      2048.0,
      16.0,
      2048.0
    ],
    "TopSurface": "Smooth"
  }
}